src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -485,13 +485,17 @@ def visit_span(self, span: Span):
             self.fcp = None
 
     def _is_blocking_render(self, span):
+        data = span.get("data", None)
+        render_blocking_status = data and data.get("resource.render_blocking_status")
```

Comment:
```suggestion
        render_blocking_status = data and data.get("resource.render_blocking_status", None)
```

protect against missing field? or nah